misc/ballerinalang-data-mapper/src/main/java/org/ballerinalang/datamapper/AIDataMapperCodeActionUtil.java

```diff
@@ -98,9 +113,9 @@
         String diagnosticMessage = diagnostic.message();
         Matcher matcher = CommandConstants.INCOMPATIBLE_TYPE_PATTERN.matcher(diagnosticMessage);
 
-        if (!(matcher.find() && matcher.groupCount() > 1)) {
-            return fEdits;
-        }
+//        if (!(matcher.find() && matcher.groupCount() > 1)) {
+//            return fEdits;
+//        }
```

Comment:
Shall we remove the commented code segment

Reply:
removed the segments